### PR TITLE
Bilibili Live helper

### PR DESCRIPTION
**Pull request by jjj201200**
Live helper
1.预约下注
2.自动签到
3.直播间左边栏在网页全屏时可见
Bilibili helper 
1.新番放送表重新排序，将近两日的番剧显示在最上方

### CODE REVIEW

File: CommentCoreLibrary.min.js [hunk 1]
```diff
@@ -1 +1,659 @@
...ransform=b(a.rY,a.rZ),c.dom.style.webkitTransform=b(a.rY,a.rZ),c.dom.style.OTransform=b(a.rY,a.rZ),c.dom.style.MozTransform=b(a.rY,a.rZ),c.dom.style.MSTransform=b(a.rY,a.rZ))}c.y=c.y,this.dispatchEvent("enterComment",c),this.runline.push(c)}},a.prototype.sendComment=function(a){console.log("CommentManager.sendComment is deprecated. Please use send instead"),this.send(a)},a.prototype.finish=function(a){this.dispatchEvent("exitComment",a),this.stage.removeChild(a.dom);var b=this.runline.indexOf(a);switch(b>=0&&this.runline.splice(b,1),a.mode){default:case 1:this.csa.scroll.remove(a);break;case 2:this.csa.scrollbtm.remove(a);break;case 4:this.csa.bottom.remove(a);break;case 5:this.csa.top.remove(a);break;case 6:this.csa.reverse.remove(a);break;case 7:}},a.prototype.addEventListener=function(a,b){"undefined"!=typeof this._listeners[a]?this._listeners[a].push(b):this._listeners[a]=[b]},a.prototype.dispatchEvent=function(a,b){if("undefined"!=typeof this._listeners[a])for(var c=0;c<this._listeners[a].length;c++)try{this._listeners[a][c](b)}catch(d){console.err(d.stack)}},a.prototype.onTimerEvent=function(a,b){for(var c=0;c<b.runline.length;c++){var d=b.runline[c];d.hold||d.time(a)}},a}();
```
Comment (zacyu) [inline CommentCoreLibrary.min.js]: Any changes to this file shall be based on https://github.com/zacyu/ABPlayerHTML5-bilibili-ver

File: bilibili_injected.js [hunk 2]
```diff
@@ -419,7 +419,7 @@
 			biliHelper.mainBlock.append(biliHelper.mainBlock.downloaderSection);
 			biliHelper.mainBlock.querySection = $('<div class="section query"><h3>弹幕发送者查询</h3><p><span></span>正在加载全部弹幕, 请稍等…</p></div>');
 			biliHelper.mainBlock.append(biliHelper.mainBlock.querySection);
-			biliHelper.helperBlock.find('.t').click(function() {
```
Comment (zacyu) [inline bilibili_injected.js]: This change is not necessary. In fact, it makes it hard for users to look up comment senders. If you do want this feature, **please make it optional** and **listen on 'click' by default**.

File: resource.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
...de{position:absolute;top:0;display:none;}#quiz_helper .cancel:hover {background-color: #F76969;}#quiz_helper .cancel:active {background-color: #F33333;}#quiz_helper .cancel {width: 200px;height: 34px;background-color: #FF4B4B;padding: 5px 15px;border: 0;border-radius: 20px;font-size: 12px;color: #fff;outline: none; -webkit-transition: all cubic-bezier(.22,.58,.12,.98) .4s;transition: all cubic-bezier(.22,.58,.12,.98) .4s;cursor: pointer;font-family: '微软雅黑','Microsoft Yahei','Microsoft Sans Serif';}#quiz_helper .success {width: 200px;height: 34px;background-color: #30BF02;padding: 5px 15px;border: 0;border-radius: 20px;font-size: 12px;color: #fff;outline: none; -webkit-transition: all cubic-bezier(.22,.58,.12,.98) .4s;transition: all cubic-bezier(.22,.58,.12,.98) .4s;cursor: pointer;font-family: '微软雅黑','Microsoft Yahei','Microsoft Sans Serif';}#quiz-control-panel .bet_toggle{position: absolute;top: 1px;right: -10px;padding:0 5px;color: #FFFFFF;background-color:#aaa;border-radius: 4px;cursor: pointer;}#quiz-control-panel .bet_toggle.on{background-color:#4fc1e9;}.bet-buy-btns{overflow:hidden;height:27px;transition: all cubic-bezier(.22,.58,.12,.98) .4s;}.bet-buy-btns.hide{height:0;}";
```
Comment (zacyu) [inline resource.js]: Please don't specify font-family here.

File: manifest.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
    },
    "content_scripts": [ {
       "all_frames": true,
-      "js": [ "bilibili_ASS_Danmaku_Downloader.js", "filename-sanitize.js", "jq.bilibili.js", "bilibili_injected.js", "ColPick.js", "CommentCoreLibrary.min.js", "ABPlayer.min.js", "cast_sender.js" , "hls.min.js" ],
+      "js": [ "bilibili_ASS_Danmaku_Downloader.js", "filename-sanitize.js", "store.min.js","jq.bilibili.js", "bilibili_injected.js","bilibili_live.js", "ColPick.js", "CommentCoreLibrary.min.js", "ABPlayer.min.js", "cast_sender.js" , "hls.min.js" ],
```
Comment (zacyu) [inline manifest.json]: store.min.js is not necessary. Chrome has it since 4.0.

**Comment by zacyu** [comment]
Thanks for your contribution. :+1: I just commented a few notable issues. Please make the changes accordingly so I can proceed to merge.

**Comment by jjj201200** [comment]
=、=直播间还有什么功能呢。。并不想加自动领瓜子。。

File: store.min.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+/* Copyright (c) 2010-2016 Marcus Westin */
```
Comment (zacyu) [inline store.min.js]: I don't think we need this file anymore.

File: bilibili_live.css [hunk 6]
```diff
@@ -0,0 +1,12 @@
+#quiz-control-panel{
```
Comment (zacyu) [inline bilibili_live.css]: Not this one either.

**Comment by zacyu** [comment]
@jjj201200 并不清楚, 我并不常看直播.

File: bilibili_live.css (hunk 6 above)
Comment (jjj201200) [inline bilibili_live.css]: TAT。。yes,yes..i always forget something

File: CommentCoreLibrary.min.js (hunk 1 above)
Comment (jjj201200) [inline CommentCoreLibrary.min.js]: no change,i just format it and scan it. i'll revert it.

Comment (zacyu) [inline CommentCoreLibrary.min.js]: Let me know when you are done and then I will merge your PR.

**Comment by jjj201200** [comment]
@zacyu Already done.

**Comment by jjj201200** [comment]
@zacyu Already done.

**Pull request merged**